js-core/homeworks/homework-5/src/main.js

```diff
@@ -61,20 +65,11 @@ console.log(moveZeroToEnd(arr2));
 
 
 function sumTwoMinimalNumber(arr) {
-    function searcheMinimalNumber(arr) {
-        let minElem = arr[0];
-        for (let i = 0; i < arr.length; i ++) {
-            if (arr[i] < minElem) {
-                minElem = arr[i];
-            }
-        }
-        return minElem;
-    }
-    let minimalNumber = searcheMinimalNumber(arr);
+    let minimalNumber = Math.min.apply(null, arr);
```

Comment:
oh :) no need to use apply here)

Do you remember about Math.min(...arr) syntax ?